packages/sanity/src/core/form/inputs/PortableText/object/modals/PopoverModal.tsx

```diff
@@ -80,42 +78,44 @@ function Content(props: PopoverEditDialogProps) {
   const [contentElement, setContentElement] = useState<HTMLDivElement | null>(null)
   const containerElement = useRef<HTMLDivElement | null>(null)
 
-  useEffect(() => {
-    // When rendered, focus on the first input element in the content
-    if (contentElement) {
-      contentElement.querySelector('input')?.focus()
-    }
-  }, [contentElement])
```

Comment:
This should rather be handled by the autoFocus prop so that it works like the other Modals we use (where the first focusable element is focused, usually the close-button).

Reply:
No longer relevant here, moved to https://github.com/sanity-io/sanity/pull/8021